### DISCUSSION TITLE
aoai access

### DISCUSSION THREAD

**Discussion by alexorgti** [karleeov/chatgpt-aoai#7] [Q&A] [answered]
how to get aoai access

**Comment by karleeov** [comment, marked as answer]
you can apply from here

https://customervoice.microsoft.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR7en2Ais5pxKtso_Pz4b1_xUOFA5Qk1UWDRBMjg0WFhPMkIzTzhKQ1dWNyQlQCN0PWcu
